content/en/docs/storage-providers/config.md

```diff
@@ -126,6 +126,23 @@ This section controls parameters for making storage and retrieval deals:
 The final value of `ExpectedSealDuration` should equal `(TIME_TO_SEAL_A_SECTOR + WaitDealsDelay) * 1.5`. This equation ensures that the miner does not commit to having the sector sealed too soon.
 {{< /alert >}}
 
+`StartEpochSealingBuffer` is minimum start epoch buffer to give time for sealing of sector with deal. In case a sector has multiple deal, this parameter enforces that sectors are packed for sealing/updating even if the nearest deal start date is close enough to the present even if there are more deals to be waited upon.
+
+```shell
+Example
+  Current Epoch:      1620712
+  Deal 1 Start Epoch: 1622712
+  Deal 2 Start Epoch: 1621712
+  Deal 3 Start Epoch: 1621212
+  Sector Number:      500
+  `StartEpochSealingBuffer` is set to `480`
+  The `WaitDealsDelay` is not over but as soon as network reaches epoch 1620732(20 epoch from now), the sector 500 will start sealing data for deal 1,2 and 3 as the buffer of 480 epochs has been reached for the deal 3(1621212-1620732=480). The new deal `Deal 4` that could have been sealed in setor 500 will now be sealed in sector 501. This will avoid delays with sealing and ensure that data is sealed before the deal start epoch.
+```
```

Comment:
```suggestion
`StartEpochSealingBuffer` allows `lotus-miner` to seal a sector before a certain epoch. For example: if the current epoch is 1000 and a deal within a sector must start on epoch 1500, then `lotus-miner` must wait until the current epoch is 1500 before it can start sealing that sector. However, if `lotus-miner` sets `StartEpochSealingBuffer` to 500, the `lotus-miner` can start sealing the sector at epoch 1000. 

If there are multiple deals in a sector, the deal with a start time closest to the current epoch is what `StartEpochSealingBuffer` will be based off. So, if the sector in our example has three deals that start on epoch 1000, 1200, and 1400, then we `lotus-miner` will start sealing the sector at epoch 500.
```